### DISCUSSION TITLE
Build for fedora 40 & 41

### DISCUSSION THREAD

**Discussion by JohnPreston** [LizardByte/community#592] [Sunshine] [answered]
### Select GitHub action

create-release-action

### Select Topic Area

Feature Request

### Body

Hello!
As ever, thank you for continuing goodness in providing new features & versions for sunshine!

As Fedora 39 has reached EOL, it might be worth adding Fedora 40 & 41 in the builds :pray: 

Thank you so very much!

**Comment by ReenigneArcher** [comment, marked as answer]
This has been done for months. https://docs.lizardbyte.dev/projects/sunshine/en/master/md_docs_2getting__started.html#fedora

**Reply by JohnPreston** [reply]
My bad!! I had gone to the releases page on GH directly 😔 //disappear
Thanks @ReenigneArcher

**Comment by JohnPreston** [comment]
@ReenigneArcher is it normal that `sunshine` is not found in the repos? I looked and don't see any sunshine RPMs for any distributions. Am I missing yet something else? :thinking:

**Reply by ReenigneArcher** [reply]
You'll need to use the beta for now

**Reply by JohnPreston** [reply]
Okay, no worries. The docs read `or` between stable and beta, but that makes sense why you'd have that.
Any reason for the package to be called `Sunshine` and not `sunshine` then?

**Reply by ReenigneArcher** [reply]
It's automated and gets the GitHub repo name.
